.github/workflows/eslint-ci.yml

```diff
@@ -2,18 +2,15 @@ name: ESLint Workflow
 
 on:
   workflow_call:
-    secrets:
-      turbo_server_token:
-        description: 'The Turborepo local server token'
-        required: true
 
 jobs:
   # Make sure eslint passes
   eslint:
     name: ESLint Check
     runs-on: ubuntu-latest
     env:
-      SERVER_TOKEN: ${{ secrets.turbo_server_token }}
+      # Use the git sha as a unique token for Turbo Repo
+      SERVER_TOKEN: ${{ github.sha }}
```

Comment:
Since we're using `github.token` directly for `repo-token`, can we do the same with `github.sha`? Without passing it as env

Reply:
We can, it's just a way to document it more clearly.